tools/importer-rest-api-specs/components/dataapigeneratorjson/templater_terraform_resource_definition.go

```diff
@@ -22,7 +24,7 @@ func buildTerraformResourceDefinition(resourceLabel string, input resourcemanage
 		},
 		Description:                  input.Documentation.Description,
 		DisplayName:                  input.DisplayName,
-		ExampleUsage:                 input.Documentation.ExampleUsageHcl,
+		ExampleUsage:                 strings.TrimPrefix(strings.TrimSuffix(input.Documentation.ExampleUsageHcl, "\n"), "\n"),
```

Comment:
I suspect, we might want to manipulate the formatting of this in the parser instead of in here. But for now I think this is fine, maybe add a `// todo` above it?